src/zh/zazhimi/src/eu/kanade/tachiyomi/extension/zh/zazhimi/Zazhimi.kt

```diff
@@ -0,0 +1,102 @@
+package eu.kanade.tachiyomi.extension.zh.zazhimi
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import keiyoushi.utils.parseAs
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+
+class Zazhimi : HttpSource() {
+
+    override val baseUrl = "https://android2024.zazhimi.net/api"
```

Comment:
This URL is user-facing. Use a different variable if you need it.
```suggestion
    override val baseUrl = "https://www.zazhimi.net"
    private val apiBaseUrl = "https://android2024.zazhimi.net/api"
```